common/core/src/main/java/com/cryart/sabbathschool/core/navigation/Destination.kt

```diff
@@ -36,3 +39,20 @@ enum class Destination(val key: String) {
         fun fromKey(type: String) = map[type]
     }
 }
+
+private const val SCHEME = "ss_app://"
+
+/**
+ * Builds a navigation [Uri] from a [Destination]
+ *
+ * Example:
+ * Passing [Destination.READ] with extras [Pair("lesson_index", "index")]
+ * will return `ss_app://read?lesson_index=index`
+ *
+ */
+fun Destination.toUri(vararg extras: Pair<String, String> = emptyArray()): Uri {
+    val query = extras.joinToString(separator = "?", prefix = "?") { pair ->
```

Comment:
I think separator should be `&`. Please add test when construct url with more than one parameter.